gripper_id:
  part_4 : "suction"
  part_5 : "suction"
  part_6 : "robotiq_gripper"
  part_7 : "suction"
  part_8 : "suction"
  part_9 : "precision_gripper_from_inside"
  part_10: "precision_gripper_from_inside"
  part_11: "suction"
  part_12: "suction"
  part_13: "suction"
  part_14: "precision_gripper_from_inside"
  part_15: "precision_gripper_from_inside"
  part_16: "precision_gripper_from_inside"
  part_17: "precision_gripper_from_outside"
  part_18: "precision_gripper_from_outside
